src/ApparentHorizons/StrahlkorperGr.cpp

```diff
@@ -651,6 +651,10 @@ std::array<double, 3> spin_vector(const double spin_magnitude,
   // Normalize spin_vector so its magnitude is the magnitude of the spin
   return spin_vector * (spin_magnitude / magnitude(spin_vector));
 }
+    
+double irreducible_mass(const double area) noexcept {
+  return sqrt(area/(16.0*M_PI));
```

Comment:
It might be a good idea to put an
```cpp
ASSERT(area > 0.0, "The area of the horizon must be greater than zero but is " << area);
```
I'm not sure if zero is allowed, but this would help when we are debugging simulations since I wouldn't be all too surprised if we end up calculating something wrong much earlier on than this function :)

Reply:
If we do this, do we still need to add a test of this assert, since we don’t do codecov in Travis? If so, I’ll have to show my students how to do this.

Reply:
I'm fine with not adding a test for the ASSERT :) Showing how to check code coverage sounds like a useful thing :)